bridges/otelzap/encoder.go

```diff
@@ -50,12 +51,9 @@ func (m *objectEncoder) AddBool(k string, v bool) {
 	m.kv = append(m.kv, log.Bool(k, v))
 }
 
-func (m *objectEncoder) AddDuration(k string, v time.Duration) {
-	// TODO
-}
-
 func (m *objectEncoder) AddComplex128(k string, v complex128) {
-	// TODO.
+	stringValue := strconv.FormatComplex(v, 'f', -1, 64)
+	m.kv = append(m.kv, log.String(k, stringValue))
```

Comment:
What do you think about representing it in a structured way as a "map"?

```suggestion
	r := log.Float64("r", real(v))
	i := log.Float64("i", imag(v))
	m.kv = append(m.kv, log.Map(k, r, i))
```

Reply:
```
func main() {
    c := complex(1, 2)
    fmt.Println(c) // Output: (1+2i)
}
```

Here, the output would be 1+2i, log.String would match this more closely than a map right. wdyt

Reply:
I just think it is better to have more accurate structure than string formatting. By formatting we can also lose more precision.